development/VehicleConfigParams.yaml

```diff
@@ -95,3 +95,50 @@ lidar_gps_drivers:
 # Parameter to enable configurable speed limit
 # Value type: Desired
 config_speed_limit: 0.0
+
+# Topics to exclude from rosbag recording
+exclude_default: true
+excluded_default_topics:
+  - /rosout(.*)
+  - (.*)/received_messages
+  - (.*)/sent_messages
+  - (.*)/scan
+
+exclude_lidar: true
+excluded_lidar_topics:
+  - /detection/lidar_detector/objects
+  - /detection/lidar_detector/objects_markers
+  - /hardware_interface/lidar/points_raw/hardware_interface/can/brake_position
+  - /hardware_interface/camera/projection_matrix/detection/lidar_detector/cloud_clusters
+  - /environment/range_vision_fusion_01/detection/lidar_detector/objects
+  - /environment/detection/fusion_tools/objects
+  - /environment/detection/fusion_tools/objects_markers
+  - /environment/detection/object_tracker/objects
+  - /environment/detection/object_tracker/objects_markers
+  - /environment/detection/objects
+  - /environment/external_object_predictions
+  - /environment/external_objects
+  - /environment/roadway_objects
```

Comment:
We don't need to exclude these last three topics. They don't contain any raw lidar/camera data.